modules/lm-coursier/src/main/scala/lmcoursier/internal/CompatParColls.scala

```diff
@@ -0,0 +1,20 @@
+package lmcoursier.internal
+
+// This is required for jscala 2.12/2.13 compatibility for parallel collections
+// (see https://github.com/scala/scala-parallel-collections/issues/22)
+private[internal] object CompatParColls {
```

Comment:
this code already exists in sbt, but is not public https://github.com/sbt/sbt/search?q=CollectionConverters